tests/data/test_subjects_dataset.py

```diff
@@ -5,6 +5,10 @@
 
 
 class TestSubjectsDataset(TorchioTestCase):
+    
```

Comment:
Could you please remove this line as well as line 76 from the other file?

Reply:
done :)